bedrock/products/templatetags/misc.py

```diff
@@ -13,6 +13,16 @@
 FTL_FILES = ["products/vpn/shared"]
 
 
+def get_available_plans(country_code, lang):
```

Comment:
Suggestion: Include "vpn" in the function name since it's specifically about plans for VPN.